mirrulations-dashboard/src/mirrdash/static/index.js

```diff
@@ -70,11 +91,13 @@ const updateClientDashboardData = () => {
         } = jobInformation;
         updateHtmlValues('jobs-waiting', num_jobs_waiting, jobs_total);
         updateHtmlValues('jobs-done', num_jobs_done, jobs_total);
+        updateCorpusProgressHtml([num_dockets_done, num_documents_done, num_comments_done], [232255, 1718669, 18072106]); //TO DO: change hard coded numbers
         // Counts
-        updateCounts("attachments-done",num_attachments_done);
-        updateCounts("comments-done",num_comments_done);
-        updateCounts("dockets-done",num_dockets_done);
-        updateCounts("documents-done",num_documents_done);
+        updateJobTypeProgress("dockets-done",num_dockets_done, 232255);
+        // TODO: figure out how to estimate number of attachments
+        updateJobTypeProgress("attachments-done",num_attachments_done, 0);
+        updateJobTypeProgress("comments-done",num_comments_done, 1718669);
+        updateJobTypeProgress("documents-done",num_documents_done, 18072106);
```

Comment:
The counts for total dockets, documents, comments should not be hardcoded but calculated every so often to reflect status of data with Regulations.gov. The numbers here were the total amounts last checked on 3/27/23